pandera/engines/pandas_engine.py

```diff
@@ -17,6 +17,7 @@
 import numpy as np
 import pandas as pd
 from packaging import version
+from pydantic import BaseModel, ValidationError
```

Comment:
Pydantic should be optional like Koalas, etc.

Reply:
pydantic's already part of the core library, I don't think we need to make it an extra

Reply:
Ah yes, I did not notice it was changed in 0.9.0.